src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
+  tempValueRobot = Object.values(robot);
+
+  for (let i = 0; i < tempValueRobot.length; i++) {
+    if (tempValueRobot.indexOf(tempValueRobot[i]) !== i) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    delete robot[key];
+  }
+
+  for (const i in tempKyesRobot) {
+    robot[tempValueRobot[i]] = tempKyesRobot[i];
```

Comment:
You're using 'tempValueRobot' as an index to 'tempKyesRobot', but earlier you derived these arrays from the same object. This might create an incorrect mapping if the values are not unique or not in the correct order. Instead, you should iterate the original robot object and swap the key-value pairs directly.